startup.py

```diff
@@ -220,7 +220,15 @@ def _find_software(self):
         """
 
         # all the executable templates for the current OS
-        executable_templates = self.EXECUTABLE_TEMPLATES.get(sys.platform, [])
+        executable_templates = self.EXECUTABLE_TEMPLATES.get(
+            "darwin"
+            if sgtk.util.is_macos()
+            else "win32"
+            if sgtk.util.is_windows()
+            else "linux2"
```

Comment:
Maybe we could fix this the other way around? The dict contains linux and we search for linux?

Reply:
I've changed it to `linux` instead of `linux2` is that what you are asking?
I'm not quite sure what you mean by other way round?